pkg/cmd/cli/cmd/deploy.go

```diff
@@ -211,6 +211,9 @@ func (o DeployOptions) RunDeploy() error {
 // deploy launches a new deployment unless there's already a deployment
 // process in progress for config.
 func (o DeployOptions) deploy(config *deployapi.DeploymentConfig, out io.Writer) error {
+	if config.Spec.Paused {
+		return fmt.Errorf("cannot deploy a paused deploymentconfig")
```

Comment:
> We shouldn't use "deploymentconfig" ever.

The thing is that we already do it. For all I know, "deploymentconfig" is valid input in the cli and "deployment config" or "deployment configuration" is not.